diagnostic.md

```diff
@@ -48,11 +48,18 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+      this.get('movies').findRecord('movie', 123);
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    In the pervious example it would look something like
+  ```html
+    {{#each movies as |movie|}}
+    <h3>{{movie.title}}</h3>
+    <h3>{{movie.rating}}</h3>
+    <h3>{{movie.length}}</h3>
+    {{/each}}
+```
```

Comment:
If this was working and included model, I could count it. `model` would be the answer we're looking for so I'll count this, you know it's the model so I will.